third_party/pytest/defs.bzl

```diff
@@ -0,0 +1,22 @@
+load("@rules_python//python:defs.bzl", "py_test")
```

Comment:
At the top of the file, please add `# Adapted from https://gist.github.com/betaboon/c1dd785b5ba468b4df4e382eafff969a` -- we should respect the original author. Same for the other file.

Reply:
I added it to the wrapper (which is now a template), but this macro I merged into the other. Just seemed more natural with the generated file. If you want me to put the comment there as well, I can sure do that, but I guess now actually not that much is left from the original.